CSharp_Course_Project/Task3/Mammal.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+
+namespace CSharp_Course_Project.Task3
+{
+    class Mammal
+    {
+        public double Age { get; private set; }
```

Comment:
Setter is excessive, could be removed since the value is changed in constructor only.

Reply:
fixed